packages/tomls/src/rotations/mainnet.toml

```diff
@@ -0,0 +1 @@
+include = ["rotations/06082025/introduce.toml"]
```

Comment:
first we need to execute "introduce.toml"
then switch from old addresses to new addresses
then we're going to add "deprecate.toml" here and this should clean up the permissions of the old addresses